frontend/src/views/components/Journey/Embed/Embed.component.jsx

```diff
@@ -8,19 +8,37 @@ import cx from 'classnames';
 import { T } from '@transifex/react';
 import { useRouter } from 'next/router';
 import { getSubdomainFromURL } from 'utilities/getSubdomain';
+import { URL_PERSISTED_KEYS } from 'state/modules/layers';
 
 // TODO: get rid of IFrame and use Map Component
 // It requires to refactor map to use redux instead of url in all cases
 // And to add separate mapper, to store all needed variables from redux
 // in url only on map page
 
+const getLayerData = (mapUrl) => {
+  const mapString = mapUrl.split('?')[1];
+  if (mapString) {
+    const mapData = mapString && qs.parse(mapString);
+    return mapData && JSON.parse(mapData.layers);
+  }
+  return null;
+};
+
+const provideAbsoluteOrRelativeUrl = (url, locale) => {
+  if (url.startsWith('http')) {
+    return url.replace('resilienceatlas.org/', `resilienceatlas.org/${locale}/`);
```

Comment:
Take this URL:
`http(s)://(subdomain.)resilienceatlas.org/(locale/)(something)`

If the current locale is Spanish, it'll be replaced by:
`http(s)://(subdomain.)resilienceatlas.org/es/(locale/)(something)`

… which is incorrect.

Do you know why we were forcing the locale? I don't think it's necessary.